libs/ui/src/components/molecules/WrappedInput/WrappedInput.tsx

```diff
@@ -9,6 +9,8 @@ export const WrappedInput = (props: Field) => {
   const registration = registerOptions
     ? register(id, registerOptions)
     : register(id);
+
+  console.log('id', id);
```

Comment:
We can probably remove this unless it's helpful for debugging while this is still under development/iteration

Reply:
Whoops.